lib/delayed_job/delayed_job_callbacks.rb

```diff
@@ -8,21 +8,20 @@ class DelayedJobCallbacks < ::Delayed::Plugin
           block.call(job)
         ensure
           duration = Time.now - start
-          Metrician.gauge("queue.process", duration) if Metrician.configuration[:queue][:process][:enabled]
-          Metrician.gauge("#{job_metric_instrumentation_name(worker)}.process", duration) if Metrician.configuration[:queue][:job_specific][:enabled]
+          Metrician.gauge("jobs.run", duration) if Metrician.configuration[:jobs][:run][:enabled]
+          Metrician.gauge("jobs.run.#{job_metric_instrumentation_name(job)}", duration) if Metrician.configuration[:jobs][:job_specific][:enabled]
```

Comment:
I updated this to reference `job` instead of `worker` because `worker` isn't a variable that exists here, and added a test.